src/confirmations_impl.h

```diff
@@ -17,11 +17,13 @@
 #include "refill_tokens.h"
 #include "redeem_token.h"
 #include "payout_tokens.h"
+#include "unblinded_tokens.h"
```

Comment:
why do we have a header include and a forward decl? Looks like only the forward decl is needed

Reply:
👍

Reply:
@bridiver My build is 5 hours away, but I removed the #include and it broke the build for Ryan, I would need to fix this once my build is complete